go/runtime/materialize.go

```diff
@@ -130,6 +131,16 @@ func (m *Materialize) RestoreCheckpoint(shard consumer.Shard) (cp pf.Checkpoint,
 		if err != nil {
 			return nil, err
 		}
+
+		// TODO(johnny): Hack to address string truncation for these common materialization connectors
```

Comment:
Very hack-y mechanism to have this take effect immediately for materialization connectors that need it. I added sqlite as an aide for end-to-end testing.